web/index.html

```diff
@@ -32,6 +32,8 @@
 </head>
 
 <body>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDn9egMgPgW78Rv0TD5a2B0lPhXbAa02Fs&libraries=places"
```

Comment:
hmmm is there any way to move this into one of the application components so we don't load maps for every page?